external/examples/writable_table/main.cpp

```diff
@@ -294,6 +305,6 @@ int main(int argc, char* argv[]) {
   }
 
   // Finally wait for a signal / interrupt to shutdown.
-  runner.waitThenShutdown();
+  runner.waitForShutdown();
```

Comment:
Ah yes, this was specifically renamed to communicate a difference of expectation.